cluster/points_writer.go

```diff
@@ -196,6 +202,13 @@ func (w *PointsWriter) WritePoints(database, retentionPolicy string, consistency
 	w.statMap.Add(statWriteReq, 1)
 	w.statMap.Add(statPointWriteReq, int64(len(points)))
 
+	if w.MaxConcurrentWrites > 0 {
```

Comment:
`/query` and `/write` have different performance characteristics so I think it makes sense to have have them separately tunable .  Many open `/query` calls is not as problematic as many `/write` ones.  `/query` has a very small payload where as `/write` can be quite large.

The `max-concurrent-queries` seems ok where it's at but moving it up to HTTP shouldn't hurt.